build_tools/rocksdb-lego-determinator

```diff
@@ -81,6 +81,7 @@ PARALLELISM="$PARALLEL_J $PARALLEL_j"
 
 DEBUG="OPT=-g"
 SHM="TEST_TMPDIR=/dev/shm/rocksdb"
+NON_SHM="TMPD=/tmp/rocksdb_test_tmp"
```

Comment:
From my reading the Makefile, I think TMPD is for parallel execution, ans TEST_TMPDIR is for non-parallel. Internally, TMPD is translated to TEST_TMPDIR when launching single test jobs in parallel execution.